pytest_django/fixtures.py

```diff
@@ -625,6 +627,11 @@ def _assert_num_queries(
     else:
         conn = connection
 
+    if using:
```

Comment:
please add a check before line 625 that both `connection` and `using` are not provided

Reply:
I thought about this originally, and decided against it at this junction, it’s a new argument that is the deciding factor. A case where both would be introduced at the same time would not be an issue for a long time.. and again, wanted to keep the changes to a minimal.

I suppose i can add a warning, which is what i intended to do in the first place… i don’t know i’m in two minds. 😇

Reply:
Ok i’ll do this! I thought you were some rando’, 😇

Reply:
Not sure if this before or after the update, but in any case, what @adamchainz meant is to add a "mutually exclusive" check before the `if connection is None: ...` part. Also we shouldn't warn, just raise an error, it's better to be strict than to silently ignore a parameter.

So something like this:

```py
from django.db import connections
from django.db import connection as default_conn

if connection is not None and using is not None:
    raise ValueError('The "connection" and "using" parameter cannot be used together')

if connection is not None:
    conn = connection
else if using is not None:
    conn = connections[using]
else:
   conn = default_conn
```